app/Models/Posts.php

```diff
@@ -30,4 +27,76 @@ public function labels()
     {
         return $this->belongsToMany('App\Models\Posts\Labels', 'posts_pivot_labels_data');
     }
+    
+    
+    // @codeCoverageIgnoreStart
```

Comment:
The whole path is ignored if I'm right so we need to delete all codecoverage comments